crypto/rand/rand_win.c

```diff
@@ -49,25 +49,25 @@ int RAND_poll(void)
     int ok = 0;
 
 # ifdef OPENSSL_RAND_SEED_RDTSC
-    rand_rdtsc();
+    rand_read_tsc(cb, arg);
 # endif
 # ifdef OPENSSL_RAND_SEED_RDCPU
-    if (rand_rdcpu())
+    if (rand_read_cpu(cb, arg))
         ok++;
```

Comment:
The same short circuiting as used in rand_unix would be beneficial here too.
The source ordering here is good.